module-platform/rt1051/src/disk_emmc.hpp

```diff
@@ -35,17 +36,22 @@ namespace purefs::blkdev
         auto get_info(info_type what, hwpart_t hwpart) const -> scount_t override;
         auto pm_control(pm_state target_state) -> int override;
         auto pm_read(pm_state &current_state) -> int override;
+        auto get_card_manufacturer() const -> std::string;
 
       private:
         auto switch_partition(hwpart_t newpart) -> int;
 
-      private:
         int initStatus;
         pm_state pmState{pm_state::active};
         mutable cpp_freertos::MutexRecursive mutex;
         std::atomic<hwpart_t> currHwPart{0};
 
         std::unique_ptr<_mmc_card> mmcCard;
         std::shared_ptr<drivers::DriverUSDHC> driverUSDHC;
+
+        /* Read from datasheets */
+        const std::map<std::uint8_t, std::string> mmcManufacturersMap{{0x15, "Samsung"},
+                                                                      {0x70, "Kingston"}, // Pure
```

Comment:
```suggestion
                                                                      {0x70, "Kingston"},
```
I would add better description of manufacturer if possible.